### DISCUSSION TITLE
System.IO.FileNotFoundException with CefSharp.Core.Runtime

### DISCUSSION THREAD

**Discussion by casetal** [cefsharp/CefSharp#4471] [General]
Hello, i've a problem with **CefSharp.Core.Runtime** on another computer.
`System.IO.FileNotFoundException: "Could not load file or assembly 'CefSharp.Core.Runtime`

Visual C++ installed, reinstalled etc... what do I fix it?

.net framework 4.7.2, cefsharp any version

Update: new error:
`System.IO.FileLoadException: "Could not load procedure imported by 'CefSharp.Core.Runtime.dll'."`

**Comment by amaitland** [comment]
> Visual C++ installed

What version? Did you install the correct architecture? 

> cefsharp any version

What version? Exact version is important.

> i've a problem with **CefSharp.Core.Runtime** on another computer.

What OS is the problem computer?

**Comment by lofti198** [comment]
Having the same issue: Windows 10 64x, .NET 4.8, version 119.4.30.8 (doing following this videoinstruction https://www.youtube.com/watch?v=si8wemHOMYA )
![image](https://github.com/cefsharp/CefSharp/assets/42960488/a5f3ad76-028e-4ae9-8baf-78fb98a8c96e)

**Reply by amaitland** [reply]
Official instructions are at https://github.com/cefsharp/CefSharp/wiki/Quick-Start

There's a working example at https://github.com/cefsharp/CefSharp.MinimalExample you can test with. Check that the minimal example works before doing anything else.

> version 119.4.30.8

The current release is `119.4.30`. There's no `119.4.30.8` build

Make sure you got the packages from `nuget.org`

**Reply by lofti198** [reply]
Thank you very much for your assistance!

**Comment by QuWuZong** [comment]
Encounter similar error messages.

Can run normally CefSharp.MinimalExample.WinForms.net472
![image](https://github.com/cefsharp/CefSharp/assets/164102702/2b6c4a18-be95-4f5e-b8b4-5c88adbd361c)

Operating system: Windows 11 version 22621.2861
Development platform: Microsoft Visual Studio Community 2022 (64 bit) - Current version 17.9.3
C++version: Microsoft Visual C++2015-2022 Redistributable (X64) -14.38.33136
CefSharp version: CefSharp WinForms 122.1.120 (installed through VS's NuGet Manager)

Project Template: Excel Add-in (Can create task pane normally)
Project Objective Framework: NET Framework 4.6.2（Simultaneously installed 4.6.2 4.7.2 4.8.1）
Project target platform: X64

Trigger code: CefSettings settings=new CefSettings();
![image](https://github.com/cefsharp/CefSharp/assets/164102702/050f77ff-41bd-45f2-a2bb-6dd58d7d2f70)

**Reply by amaitland** [reply]
> Project Template: Excel Add-in (Can create task pane normally)

Running `CefSharp` as a plugin (a plugin to anything) is problematic as it can only be run in the default `AppDomain` (a limitation of `VC++`).

You can run CefSharp in a seperate process to get around this limitation. There's a generic example at  https://github.com/cefsharp/CefSharp.OutOfProcess

Alternatively look for a different web browser.
